src/styles/main.scss

```diff
@@ -1,3 +1,281 @@
 body {
   margin: 0;
 }
+
+$maincolor: #060b35;
+$secondarycolor:#00ACDC;
+$font_family:"Roboto", sans-serif;
+
+@mixin Nav_values {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+  cursor: pointer;
+}
```

Comment:
remove this mixing and add these styles to .nav__link (remove all other duplicates for old classes)